integration/inspektor-gadget/integration_test.go

```diff
@@ -909,6 +911,57 @@ func TestOpensnoop(t *testing.T) {
 	RunCommands(commands, t)
 }
 
+func TestNetworkGraph(t *testing.T) {
+	ns := GenerateTestNamespaceName("test-networkgraph")
+
+	t.Parallel()
+
+	networkGraphCmd := &Command{
+		Name:         "StartNetworkGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace network -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &networkTypes.Event{
+				Event: eventtypes.Event{
+					Type: eventtypes.NORMAL,
+					CommonData: eventtypes.CommonData{
+						Namespace: ns,
+						Pod:       "test-pod",
+					},
+				},
+				PktType:    "OUTGOING",
+				Proto:      "tcp",
+				Port:       443,
+				RemoteKind: "other",
+			}
+
+			normalize := func(e *networkTypes.Event) {
+				e.Node = ""
+				e.Container = ""
+				e.IP = ""
+				e.PodHostIP = ""
+				e.PodIP = ""
+				e.PodOwner = ""
+				e.PodLabels = nil
+				e.RemoteOther = ""
+				e.Debug = ""
+			}
+
+			return ExpectEntriesToMatch(output, normalize, expectedEntry)
+		},
+	}
+
+	commands := []*Command{
+		CreateTestNamespaceCommand(ns),
+		networkGraphCmd,
+		BusyboxPodRepeatCommand(ns, "wget -q -O /dev/null https://kinvolk.io"),
```

Comment:
You could use the same test-pod used in [the guide](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/guides/trace/network-graph.md#on-kubernetes) so that you can verify also the IP "1.1.1.1"